metadata-ingestion/src/datahub/ingestion/source/bigquery_v2/usage.py

```diff
@@ -328,6 +332,22 @@ def usage_statistics(self, top_n: int) -> Iterator[UsageStatistic]:
                 column_freq=json.loads(row["column_freq"] or "[]"),
             )
 
+    def delete_original_read_events_for_view_query_events(self) -> None:
+        # CAUTION: Be careful when updating this query
+        # Ideally we should clean_active_object_cache python cache here
```

Comment:
We flush the cache on `sql_query` so any query should be safe, right?

Reply:
right.